Examples/RegistrationITKv4/DeformableRegistration4.cxx

```diff
@@ -437,6 +437,7 @@ main(int argc, char * argv[])
                                             CoordinateRepType>;
 
   /** Create an setup displacement field generator. */
+  /** @ITKStartGrouping */
```

Comment:
This is a comment in an implementation .cxx file. Doxygen grouping doesn't make sense here.

Reply:
The grouping is the automatic grouping as done by the original script as well (the, generated, settings file specified to run the filter over the `*.h` and `*.cxx` files), there might be instances where the grouping was done but is not wanted or required (these cases have to be investigated and a decision, best after the integration of this PR as follow up work)